### PR TITLE
Emerge from the "beta" stage

### PR DESCRIPTION
**Pull request by fghaas**
This is a series of patches that should land in one go, when we take the site out of beta.

### CODE REVIEW

**Comment by fghaas** [comment]
Note to self, this is a little odd. #125 has now landed, meaning everything up to 0805074c31b201b7cd1c5b553bc4f2a72e75861c is already in `main`, leaving only 2 commits in this patch series — but the _Commits_ tab still shows 5.

**Comment by fghaas** [comment]
@colder-is-better This is still marked Draft as a precaution against a premature merge, but I would still appreciate a review (at your convenience; no rush here).

File: docs/index.md [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
-products and services. If you find any published documentation that is
-inaccurate or not in line with functionality as you observe it on
-{{brand}}, we would very much appreciate if you filed
-[a documentation bug]({{config.repo_url}}/issues).
+All content on this site is licensed under the [{{legal_docs.cc_by_sa.short_name}}]({{legal_docs.cc_by_sa.url}}) license.
 
-However, the documentation on this site may be incomplete, meaning
-that it does not yet cover *all* functionality available in
-{{brand}}.
+We host our site content [on GitHub]({{config.repo_url}}), and if you would like to help by contributing a patch, [here’s how](contrib/index.md)!
```
Comment (colder-is-better) [inline docs/index.md:10]:
```suggestion
We host our site content [on GitHub]({{config.repo_url}}), and if you would like to help by contributing a patch, [here’s how you can do it](contrib/index.md)!
```

**Review by colder-is-better — COMMENTED** [review]
Looking good --- I only have a pico-suggestion!

File: docs/index.md (hunk 1 above)
Comment (fghaas) [reply docs/index.md:10]: Hmmm. I'm not sure if that's an improvement? :)

"..., here's how you do that!" would be a change I could settle for.

Comment (colder-is-better) [reply docs/index.md:10]: I think a verb is missing, hence the "do". And yes, "that" is better than "it".

Comment (fghaas) [reply docs/index.md]: Done.

**Review by colder-is-better — DISMISSED** [review]

**Comment by fghaas** [comment]
@colder-is-better I have now taken this out of the Draft stage, and rebased on current `main`. I intend to merge this later today; please re-approve at your convenience. Thanks!

**Review by colder-is-better — APPROVED** [review]

**Pull request merged**